internal/mode/static/state/graph/nginxproxy_test.go

```diff
@@ -266,6 +266,11 @@ func TestValidateNginxProxy(t *testing.T) {
 						},
 					},
 					IPFamily: helpers.GetPointer[ngfAPI.IPFamilyType](ngfAPI.Dual),
+					RewriteClientIP: &ngfAPI.RewriteClientIP{
+						SetIPRecursively: helpers.GetPointer(true),
+						TrustedAddresses: []ngfAPI.TrustedAddress{"2001:db8:a0b:12f0::1/32", "0.0.0.0/0"},
```

Comment:
Can we change "0.0.0.0/0" to an actual CIDR to make this test more realistic? "0.0.0.0/0" allows all CIDRs so it wouldn't make sense in a list like this